src/components/PeoplePage.tsx

```diff
@@ -1,35 +1,176 @@
+    });
+  };
 
-              <p data-cy="peopleLoadingError">Something went wrong</p>
+  const SORT_LIST = ['Name', 'Sex', 'Born', 'Died'];
+
+  return (
+    <>
+      <h1 className="title">People Page</h1>
+      {hasError ? (
+        <tr>
+          <td
+            className="has-text-danger"
+            data-cy="peopleLoadingError"
+          >
+            Something went wrong
+          </td>
+        </tr>
+      ) : (
+        <div className="box table-container">
+          {isLoading ? (
+            <Loader />
+          ) : (
+            <div className="block">
+              <div className="columns is-desktop is-flex-direction-row-reverse">
+                <div className="column is-7-tablet is-narrow-desktop">
+                  <PeopleFilters />
+                </div>
 
-              <p data-cy="noPeopleMessage">
-                There are no people on the server
-              </p>
+                <div className="column">
+                  <table
+                    data-cy="peopleTable"
+                    className="table is-striped is-hoverable
+                            is-narrow is-fullwidth"
+                  >
+                    <thead>
+                      <tr>
+                        {SORT_LIST.map(sortItem => (
+                          <th key={sortItem}>
+                            <span className="is-flex is-flex-wrap-nowrap">
+                              {sortItem}
+                              <SearchLink
+                                params={getSortingParams(
+                                  sortItem.toLowerCase(),
+                                )}
+                              >
+                                <span className="icon">
+                                  <i
+                                    className={
+                                      getSortIconClass(sortItem)
+                                    }
+                                  />
+                                </span>
+                              </SearchLink>
+                            </span>
+                          </th>
+                        ))}
+                        <th>Mother</th>
+                        <th>Father</th>
+                      </tr>
+                    </thead>
 
-              <p>There are no people matching the current search criteria</p>
+                    <tbody>
+                      {people.length === 0 ? (
+                        <tr>
+                          <td colSpan={6} data-cy="noPeopleMessage">
+                            There are no people on the server
+                          </td>
+                        </tr>
+                      ) : (
+                        visiblePeople.map((person) => {
+                          const hasMother = people
+                            .find(({ name }) => name === person.motherName)
+                            || null;
+                          const hasFather = people
```

Comment:
- Method `find` return an element, not boolean, so the names should be mother/father.